pytorch_lightning/strategies/deepspeed.py

```diff
@@ -334,7 +427,19 @@ def _load_config(self, config):
                 config = json.load(f)
         return config
 
-    def setup_distributed(self):
+    def set_world_ranks(self) -> None:
+        if self.cluster_environment is None:
+            return
+        self.cluster_environment.set_global_rank(self.node_rank * self.num_processes + self.local_rank)
+        self.cluster_environment.set_world_size(self.num_nodes * self.num_processes)
+        rank_zero_only.rank = self.cluster_environment.global_rank()
+
+    def _configure_launcher(self) -> None:
+        self._launcher = _SubprocessScriptLauncher(self.cluster_environment, self.num_processes, self.num_nodes)
+        if not self.cluster_environment.creates_processes_externally:
+            self._rank_0_will_call_children_scripts = True
+
+    def setup_environment(self) -> None:
```

Comment:
Let's follow DDP structure with `setup_environment` and `setup_distributed` for consistency.